hoot-core/src/main/cpp/hoot/core/auth/HootServicesLoginManager.cpp

```diff
@@ -42,6 +42,11 @@
 #include <boost/property_tree/json_parser.hpp>
 #include <boost/foreach.hpp>
 
+// Qt
+#include <qnetworkcookie.h>
+#include <qurl.h>
+#include <qurlquery.h>
+
```

Comment:
I think for code consistency we should use `#include <QUrlQuery>` and the like instead of the `.h` files directly. 
 https://code.woboq.org/qt5/qtbase/include/QtCore/QUrlQuery.html tells me that `QUrlQuery` only contains the following:
```
#include "qurlquery.h"
```
So like I said, just for consistency since we do it everywhere else in the code.

Reply:
Fixed